app/views/articles/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Article</h1>
+<h2>Article By - <%= @user.email %> </h2>
+<p>
+	<strong>Title:</strong><br>
+	<%= @article.title %>
+</p>
```

Comment:
Here you are following no convention. After <p> tag somewhere you are putting tabs and sometimes you are not. We usually don't.